_posts/2018-09-28-3D-Printing-Intro.md

```diff
@@ -0,0 +1,400 @@
+
+
+## Getting started with 3D printing
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/3D-print-fail.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+Getting started with 3D printing yourself is not easy, it can be very daunting.
+There are a lot of things you need to take into account
+
+Since it is still relatively expensive, you want to ask yourself these questions:
+
+- What type of 3D printing do I want?
+- What is the best (price/quality) printer available?
+- Will I use it enough to justify the purchase?
+- You could also just order a printed model online!
+
+Getting things right is hard:
+
+- Printer calibration
+- Model slicing & print settings
+
+⇒ 3D printing remains an intensive trial and error process!
+Only begin if you are willing to invest the necessary time into it.
+If at first you don't succeed, try again, and again, and again, and again!
+
+Thankfully the online hobby community is very large and generally very helpful.
+You will need to model your object, or download it from a maker website.
+You will need to slice your model with setting for your specific printer, and adjust these settings based on the quality and speed you want.
+
+*Online resources:*
+
+- Youtube channels:
+  - [Maker's Muse](https://www.youtube.com/user/TheMakersMuse)
+  - [3D Printing Nerd](https://www.youtube.com/channel/UC_7aK9PpYTqt08ERh1MewlQ)
+  - [Make Anything](https://www.youtube.com/channel/UCVc6AHfGw9b2zOE_ZGfmsnw)
+- Forums:
+  - [3D Hubs](https://www.3dhubs.com/)
+  - [Reddit 3D printing](https://www.reddit.com/r/3Dprinting/)
+
+  These YouTube channels and forums are an invaluable resource when getting into 3D printing, they contains loads of information, do's and don'ts.
+  By watching and reading videos and articles you can prevent mistakes and gain insights faster in how 3D printing works and what is possible.
+
+*Maker websites:*
+
+- [Thingiverse](https://www.thingiverse.com/)
+- [Tinkercad](https://www.tinkercad.com/#/)
+- [MyMiniFactory](https://www.myminifactory.com/)
+
+These websites provide vast libraries of models and some even offer an online modeling tool.
+Before diving deep into your modeling software check if someone else had the same idea and created the object you want.
+
+*Modeling software:*
+
+- Modeling software:
+  - [3DS Max](https://www.autodesk.be/nl/products/3ds-max/overview)
+  - [Maya](https://www.autodesk.be/nl/products/maya/overview)
+  - [Blender](https://www.blender.org/)
+- Card software:
+  - [Fusion 360](https://www.autodesk.com/products/fusion-360/overview)
+  - [Autocad](https://www.autodesk.be/nl/products/autocad/overview)
+  - [Inventor](https://www.autodesk.be/nl/products/inventor/overview)
+
+These software suites let you create and export models.
+These programs are commonly easy to pickup (except for the CAD softwares) but are extremely hard to master!
+There are also many alternatives available, but these are the most common ones.
```

Comment:
These software suites let you create and export models.
These programs are commonly easy to pickup (except for the CAD softwares) but are extremely hard to master!
=>
These software suites let you create and export models.
All of these programs are easy to pick up (except for CAD software) but extremely hard to master.